llvm/lib/CodeGen/AsmPrinter/DwarfUnit.cpp

```diff
@@ -1379,6 +1379,9 @@ void DwarfUnit::applySubprogramAttributes(const DISubprogram *SP, DIE &SPDie,
   if (!SP->getTargetFuncName().empty())
     addString(SPDie, dwarf::DW_AT_trampoline, SP->getTargetFuncName());
 
+  if (SP->isDebugTransparent())
```

Comment:
```suggestion
    addString(SPDie, dwarf::DW_AT_trampoline, SP->getTargetFuncName());
  else if (SP->isDebugTransparent())
```
You don't want to emit the same attribute twice in the same record.